datastore/indexmanager.go

```diff
@@ -12,6 +12,8 @@ import (
 
 	"github.com/jeffpierce/cassabon/config"
 	"github.com/jeffpierce/cassabon/logging"
+
+	"github.com/otium/queue"
```

Comment:
Go convention states that this import should be above our own package imports, still on its own line.